Snowplow iOSTests/TestRequest.m

```diff
@@ -84,28 +84,6 @@ - (void)testRequestSendWithGet {
     XCTAssertEqual([tracker.emitter getDbCount], 0);
 }
 
-- (void)testRequestSendWithBadUrl {
```

Comment:
Where did this test go? Whilst I'm not sure it seems particularly valuable this seems like an odd time to delete it.

Reply:
This is a very flaky test, especially in the last Xcode 12.3 version.
I removed it because a larger refactoring of this unit test is present in the iOS tracker v.2.0.
So the test is not actually lost, but it's just too hard to fix for this little patch version.
https://github.com/snowplow/snowplow-objc-tracker/pull/560/commits/8f232ac6ba4610f84e6b3155ef8846c6b2ec3e7d